rbac/app/address.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
```

Comment:
Regarding the file name: What kind of address is this? Block address? transaction address? I see user_address, and am wondering if this could benefit from a more meaningful name.